modules/luci-mod-network/htdocs/luci-static/resources/view/network/interfaces.js

```diff
@@ -909,6 +909,11 @@ return view.extend({
 						_('Forward DHCPv6 messages between the designated master interface and downstream interfaces.'));
 					so.value('hybrid', _('hybrid mode'), ' ');
 
+					so = ss.taboption('ipv6', form.Value, 'dhcpv6_pd_min_len', _('<abbr title="Prefix Delegation">PD</abbr> minimum length'),
+						_('Configures the minimum delegated prefix length assigned to a requesting downstream router, potentially overriding a requested prefix length. If left unspecified, the device will assign the smallest available prefix greater than or equal to the requested prefix.'));
+					so.optional = true;
```

Comment:
Think you can drop this as it's true by default?

Reply:
Sure.  (I was following the other examples in the file, such as line 687 for DHCP range `start` value.)

Reply:
Just to be sure, remove the optional line and try to save a conf from luci with that value empty

Reply:
Yup, saving with the form empty, the written config section omits the setting.  Saving with it filled in, it writes the setting with value.  saving again with the form field empty, the written config again omits the setting.